modules/ROOT/partials/configuration/text_patterns.adoc

```diff
@@ -87,30 +98,33 @@ NOTE: Block patterns are only executed on *Enter*, *not* on pressing the *spaceb
 tinymce.init({
   selector: 'textarea', // change this value according to your HTML
   // The `lists` plugin is required for list-related text patterns
+  // The trigger property can be configured to use either space or enter.
   plugin: 'lists',
   text_patterns: [
-    { start: '#', format: 'h1' },
-    { start: '##', format: 'h2' },
-    { start: '###', format: 'h3' },
-    { start: '####', format: 'h4' },
-    { start: '#####', format: 'h5' },
-    { start: '######', format: 'h6' },
-    { start: '* ', cmd: 'InsertUnorderedList' },
-    { start: '- ', cmd: 'InsertUnorderedList' },
-    { start: '1. ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'decimal' } },
-    { start: '1) ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'decimal' } },
-    { start: 'a. ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'lower-alpha' } },
-    { start: 'a) ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'lower-alpha' } },
-    { start: 'i. ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'lower-roman' } },
-    { start: 'i) ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'lower-roman' } }
+    { start: '*', end: '*', format: 'italic' },
+    { start: '**', end: '**', format: 'bold' },
+    { start: '#', format: 'h1', trigger: 'space' },
+    { start: '##', format: 'h2', trigger: 'space' },
+    { start: '###', format: 'h3', trigger: 'space' },
+    { start: '####', format: 'h4', trigger: 'space' },
+    { start: '#####', format: 'h5', trigger: 'space' },
+    { start: '######', format: 'h6', trigger: 'space' },
+    { start: '*', cmd: 'InsertUnorderedList', trigger: 'space' },
+    { start: '- ', cmd: 'InsertUnorderedList', trigger: 'enter' },
+    { start: '1.', cmd: 'InsertOrderedList', value: { 'list-style-type': 'decimal' }, trigger: 'space' },
+    { start: '1) ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'decimal' }, trigger: 'enter' },
+    { start: 'a.', cmd: 'InsertOrderedList', value: { 'list-style-type': 'lower-alpha' }, trigger: 'space' },
+    { start: 'a) ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'lower-alpha' }, trigger: 'space' },
```

Comment:
```suggestion
    { start: 'a) ', cmd: 'InsertOrderedList', value: { 'list-style-type': 'lower-alpha' }, trigger: 'enter' },
```

Reply:
i assume this one is meant to be an `enter` trigger as the start parameter ends with a `space`.

Reply:
Right.